app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id])
```

Comment:
toán tử `||=` có tác dụng gì

Reply:
nếu biến @current_user đã tồn tại thì lấy luôn giá trị đó, còn nếu không thì @current_user = User.find_by(id: session[:user_id])